### DISCUSSION TITLE
Can't Create a New Module

### DISCUSSION THREAD

**Discussion by ballcoach12** [testcontainers/testcontainers-go#2816] [Q&A] [answered]
I am not sure what I'm doing wrong, but I cannot seem to successfully create a new module.  I am issuing the following command:

` go run . new module --name vfs --image "myartifactory/vfs:1.0.6-24276.75251f4.develop" --title VFS`

However, no matter what, I get 

`>> error checking generated code: exit status 1`

No other error information.  I'm running Go 1.22.2, and this happens on Linux and Win11.

**Comment by mdelapenya** [comment, marked as answer]
Oh I think I know what happens: you probably will see generated code, but there are some imports missing. There is a template probably missing the "testcontainers" core package import in both the example and test files. Could you verify that?

To fix that, in your generated code, using your IDE, save those two files, to auto-import the packages.

Also please submit a PR updating the templates to include the packages if possible 🙏

**Reply by ballcoach12** [reply]
You are correct.  I'll look into that PR and make the change as soon as I get a free minute or two.  Thanks!

**Reply by mdelapenya** [reply]
Thank you so much 🙇🏻
